es-vue-base/src/lib-components/EsFormMsg.vue

```diff
@@ -23,57 +22,69 @@ import {
 } from 'bootstrap-vue';
 import IconCircleAlert from '../lib-icons/icon-circle-alert.vue';
 import IconCircleCheck from '../lib-icons/icon-circle-check.vue';
+import IconInfo from '../lib-icons/icon-info.vue';
 
 export default {
     name: 'EsFormMsg',
     components: {
         BAlert,
         IconCircleAlert,
         IconCircleCheck,
+        IconInfo,
     },
     props: {
-        message: {
-            type: String,
-            default: '',
-            required: false,
+
+        show: {
+            type: Boolean,
+            default: false,
         },
+
         timeout: {
             type: Number,
-            default: 20,
+            default: 5,
```

Comment:
One thing I realized - we should probably keep this default at 20 seconds for the EsFormMsg component itself, otherwise forms in the real world that use this might have their errors disappear before customers have a chance to fully read them.